File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.java
class Solution {
    public String solution(String s) {
        
        int num = s.length()/2;
        
        String answer = "";
        
        if(s.length()%2 == 0){
            answer = s.substring(num-1, num+1);
        }else{
            answer = s.substring(num, num+1);
        }
        
        return answer;
    }
}

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.js
function solution(s) {
    
    var num = Math.floor(s.length/2);
    
    var answer = '';
    
    if(s.length%2 == 0){
        answer = s.substring(num-1, num+1);
    }else{
        answer = s.substring(num, num+1);
    }
    
    return answer;
}